atomic_dbus.py

```diff
@@ -374,6 +378,40 @@ def _ScanSetup(self, scan_targets, scanner, scan_type, rootfs, _all, images, con
     def Scan(self, scan_targets, scanner, scan_type, rootfs, _all, images, containers):
         return self._ScanSetup(scan_targets, scanner, scan_type, rootfs, _all, images, containers).scan()
 
+    @slip.dbus.polkit.require_auth("org.atomic.read")
+    @dbus.service.method("org.atomic", in_signature='ssss', out_signature= 's')
+    # sudo busctl --system call org.atomic /org/atomic/object org.atomic ActiveScans
+    # Only support scanning one image at a time via the async method
+    # Same with rootfs
+    def ScanAsync(self, scan_id, scanner=None, scan_type=None, rootfs=None):
```

Comment:
I understand. The problem is that D-Bus itself doesn't have the concept of `None`. This seems to work in the python-dbus, but not from other languages. If you change `None` to the empty string as a default (which should be fine, as empty strings aren't valid ids or paths), it will work everywhere.